app/assets/stylesheets/components/_theme.css.scss

```diff
@@ -146,6 +148,7 @@ md-input-group {
 md-toolbar {
   @include gamma;
   padding: 16px;
+  max-height: $toolbar-max-height;
```

Comment:
Did you consider to use `height` instead? Or can the toolbar be smaller than `60px`?